### PR TITLE
Add galaxy demo for Falcon7b

### PR DESCRIPTION
**Pull request by skhorasganiTT**
### Ticket
N/A

### Problem description
N/A

### What's changed
- Created Falcon7b galaxy demo (data parallel) with corresponding input data json, expected greedy output json, README
- Disabled printing per token for multi-device runs of the demo
- Corrected comment in conftest.py specifying the device_mesh grid for TG
- Added codeowners for models/demos/tg/falcon7b
- Added tests to tg demo pipeline
- Added updated benchmark csv targets for Falcon7b-decode to WH/T3K/galaxy demos
- Changed schedule of tg demo pipeline to Monday/Wednesday/Friday (12am UTC)

### Checklist
- [x] Post commit CI passes
- [x] Blackhole Post commit (if applicable)
- [x] Model regression CI testing passes (if applicable)
- [x] New/Existing tests provide coverage for changes

### CODE REVIEW

**Comment by skhorasganiTT** [comment]
Post-commit model tests: https://github.com/tenstorrent/tt-metal/actions/runs/10289277133

**Review by cfjchu — APPROVED** [review]

**Review by tt-rkim — APPROVED** [review]

**Comment by tt-rkim** [comment]
Not putting this into CI?

**Comment by skhorasganiTT** [comment]
@tt-rkim Added CI tests (verifying now, will not merge until passes) and also updated codeowners

**Comment by skhorasganiTT** [comment]
@ttmchiou @tt-rkim I also changed the schedule of the TG demo pipeline to Mon/Wed/Fri (12am UTC) since once a week is insufficient for catching regressions. If that is not alright, please let me know.

File: models/demos/t3000/falcon7b/demo_t3000.py
Comment (ttmchiou) [inline models/demos/t3000/falcon7b/demo_t3000.py:68]: do we need to test t3k demos if we made this change?

Comment (skhorasganiTT) [reply models/demos/t3000/falcon7b/demo_t3000.py:68]: Ran t3k and single-card demos, both are passing.

**Review by uaydonat — APPROVED** [review]

**Comment by skhorasganiTT** [comment]
Skipping TG tests on CI due to ND galaxy issues (#11258, #11254)

**Comment by tt-rkim** [comment]
Force merged because @skhorasganiTT said this is urgent adn @ttmchiou on vacation

**Pull request merged**
